--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -431,7 +431,7 @@
       if(CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG")
         set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDebug)
       elseif(MSVC_IDE)
-        set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+        set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded)
       else()
         set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded)
       endif()
--- a/sharedlib/CMakeLists.txt
+++ b/sharedlib/CMakeLists.txt
@@ -16,7 +16,7 @@
     if(CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG")
       set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDebugDLL)
     elseif(MSVC_IDE)
-      set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
+      set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
     else()
       set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
     endif()
